src/services/saveImagesToUSB.ts

```diff
@@ -0,0 +1,49 @@
+import { IService } from '../types';
+import { readdirSync, existsSync, mkdirSync, rename, statSync, readdir } from 'fs';
+import { USB_WRITE_PATH } from 'config';
+
+export const SaveImagesToUSB: IService = {
```

Comment:
are we saving images to the usb, or are we moving images to a new directory?  It's really important that we are clear and accurate with our function names and descriptions.